packages/graphback-runtime-mongodb/src/queryBuilder.ts

```diff
@@ -115,7 +115,12 @@ function traverse(filter: any): any {
                     filter[key] = [filter[key]];
                 }
             }
-
+            if (["createdAt", "updatedAt"].includes(key)) {
+                const entries = Object.entries(filter[key]).map((entry: [string, string])=> {
+                    return [entry[0], parseInt(entry[1], 10)]
+                })
+                filter[key] = Object.assign({}, ...Array.from(entries, ([k, v]: [string, any]) => ({[k]: v}) ));
```

Comment:
I assume here we want to create the object from entries, if so here, we can achieve this with this

```suggestion
                filter[key] = Object.fromEntries(entries);
```

Reply:
I tried that too but I got `Property 'fromEntries' does not exist on type 'ObjectConstructor'`

Reply:
Okay, you must have at least node 12.0.0.

Reply:
Anyway, it was a minor suggestion. You can ignore it for now and focus on the big stuffs ;-)

Reply:
I think we need to add [es2019.object](https://github.com/microsoft/TypeScript/blob/master/lib/lib.es2019.object.d.ts) to tsconfig.lib to make it work

Perhaps here would be fine: https://github.com/aerogear/graphback/blob/db8d973108a973665488416dd761a2f68a4b8c41/packages/graphback-runtime-mongodb/tsconfig.json#L10